Tasks/DownloadJenkinsArtifacts/task.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Per our discussion,, we should enhance existing "Download Jenkins artifact" task instead of adding another task.